apps/molgenis-components/src/components/layout/Molgenis.vue

```diff
@@ -66,12 +66,17 @@ export default {
         {
           label: "Schema",
           href: "schema",
-          role: "Manager",
+          role: "Viewer",
         },
         {
           label: "Up/Download",
           href: "updownload",
-          role: "Editor",
+          role: "Viewer",
```

Comment:
updown Viewer ?

Reply:
yes, it is allowed to download, not to upload. So this is useful for a viewer also

Reply:
check